json-schema-bundler-cli.mjs

```diff
@@ -64,7 +67,18 @@ let schema;
 
 try {
     if (argv.d) {
-        schema = await $RefParser.dereference(input);
+        let circular = true; // default value for `dereference`
+        switch (argv.c) {
+            case 'false':
+                circular = false;
+                break;
+            case 'ignore':
+                circular = argv.c;
+                break;
+            default:
+                break;
+        }
```

Comment:
I think the switch is a bit verbose in this case and a else if would be fine in this case :)

```suggestion
        let circular = true;
        if (argv.c === 'false') {
            circular = false;
        } else if (argv.c === 'ignore') {
            circular = 'ignore';
        }
```